### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Lmasniuk**

### CODE REVIEW

File: source/docs/guides/credential_on_file/index.md [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+to store the cardholder's payment credentials for the purpose of charging the cardholder again over regular intervals, up to a specified end date.  
+An example of this type is when the cardholder has purchased a fridge and is paying for it in installments.  This type will be the first payment in 
+the set of installment payments, and it can be either a purchase or a pre-auth transaction.
+
+Since this is the first transaction in a set of transactions, no series ID is required, and a new series ID will be returned back in the response message.
+
+#### Request 1
+
+```curl
+curl -X POST https://test01-api.na.bambora.com/v1/payments
```
Comment (dan-sparkes-bambora) [inline source/docs/guides/credential_on_file/index.md]: maybe this should be prod?

File: source/docs/guides/credential_on_file/index.md [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+        "avs": {
+            "id": "0",
+            "message": "Address Verification not performed for this transaction.",
+            "processed": false
+        }
+    },
+    "links": [
+        {
+            "rel": "void",
+            "href": "https://test01-api.na.bambora.com/v1/payments/10000589/void",
```
Comment (dan-sparkes-bambora) [inline source/docs/guides/credential_on_file/index.md]: also should maybe be prod?

**Comment by dan-sparkes-bambora** [comment]
There were more references to test01, but I figured I didn't need to point out them all

**Comment by aurorawalker** [comment]
Closing this pull request as the changes associated with PAY-2394 are not ready and are blocking production deploys.

I will create a new pull request that reverts the commits associated with PAY-2394, and set up a new PR ready to re-introduce PAY-2394 when it is ready. I will also be sure to transfer the comments Dan put on here, in regards to references to test01 on the new pull requet.

**Pull request closed without merging**